### ISSUE TITLE
How to do shape inference, wrong doc and missing APIs

### ISSUE THREAD

**Issue by FrancescoSaverioZuppichini** [microsoft/onnxruntime#13201]
### Describe the issue

[Doc](https://onnxruntime.ai/docs/performance/quantization.html) states to use 

```
python -m onnxruntime.quantization.shape_inference --help
```

It doesn't exist

```
 No module named onnxruntime.quantization.shape_inference
```

### To reproduce

Install `onnxruntime`, follow the doc for quantization

### Platform

Linux

### OS Version

20.04

### ONNX Runtime Installation

Released Package

### ONNX Runtime Version or Commit ID

1.12.1

### ONNX Runtime API

Python

### Architecture

AMD

### Execution Provider

Default CPU

**Comment by yufenglee** [comment]
The API will be available in 1.13 or nightly build. The shape inference is done automatically for standard ONNX model. The new API is mainly for model with operators from domain com.microsoft. @chenfucn .

**Comment by chenfucn** [comment]
Should be available now that 1.13 was released.
